src/NopCommerce.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14

```

Comment:
Ugh must of missed it in my commit

On Jul 22, 2016 10:27 PM, "Reza Ahmadi" <notifications@github.com> wrote:

> In src/NopCommerce.sln
> <https://github.com/nopSolutions/nopCommerce/pull/1596#discussion_r71966120>
> :
>
> > @@ -1,7 +1,7 @@
> >  ﻿
> >  Microsoft Visual Studio Solution File, Format Version 12.00
> > -# Visual Studio 14
>
> Why did you change the sln file?!!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nopSolutions/nopCommerce/pull/1596/files/1f48dfd3308978a67c1e949d639e3a1ba5a4e6a4#r71966120>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmVG05-k8sgJzF30-CbNkycUhvb3QK9ks5qYYoRgaJpZM4JS90X>
> .
>